server.py

```diff
@@ -140,6 +140,15 @@ async def upload_image(request):
             else:
                 return web.Response(status=400)
 
+        @routes.get("/output/images")
+        async def get_output(request):
+            output_dir = os.path.join(os.path.dirname(os.path.realpath(__file__)), "output")
+
+            if not os.path.exists(output_dir):
+                return web.Response(status=404)
+
```

Comment:
```suggestion
            os.makedirs(output_dir, exist_ok=True)
```
You wouldn't need to include the empty output folder, either.